File: ComputationalGeometry/pagina01/1039/1039.js
const fs = require("fs")
const input = fs.readFileSync('/dev/stdin', 'utf8').split("\n");

function fire_flowers(data) {
    const [ r1, x1, y1, r2, x2, y2 ] = data.split(' ').map(Number)
    const distancia_do_centro = Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2))
    
    distancia_do_centro <= r1 - r2 ? console.log('RICO') : console.log('MORTO')
}

for(let i = 0; i < input.length; i++){
    fire_flowers(input[i])
}

File: ComputationalGeometry/__1000__/1039/1039.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool fire_flowers(int r1, int x1, int y1, int r2, int x2, int y2){
    double distancia = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));
    return distancia <= (r1 - r2);
}

int main() {
    int r1, int x1, int y1, int r2, int x2, int y2

    while ( cin >> r1 >> x1 >> y1 >> r2 >>  x2 >> y2 ){
        if(fire_flowers(r1, x1, y1, r2, x2, y2)){
            cout << "RICO" << endl;
            continue;
        }

        cout << "MORTO" << endl;
    }
    return 0;
}